pact/fixed-quote-royalty-policy.pact

```diff
@@ -124,8 +128,11 @@
             (recipient-guard:guard (at 'recipient-guard spec))
             (recipient-details:object (fungible::details recipient))
             (sale-price:decimal (* amount price)) )
-      (fungible::enforce-unit (* sale-price royalty-rate))
-      (enforce (< 0.0 price) "Offer amount must be positive")
+      (if
+        (= royalty-rate 0.0)
+        (fungible::enforce-unit sale-price)
```

Comment:
This needs to not be conditional; it's the following test that needs the condition

Reply:
Removed the conditionals, and floored the `royalty-payout`